javatools_backend/src/main/java/org/xvm/xtc/ast/InvokeAST.java

```diff
@@ -4,11 +4,12 @@
 import org.xvm.xtc.*;
 import org.xvm.xtc.cons.Const;
 import org.xvm.xtc.cons.MethodCon;
+import org.xvm.xec.ecstasy.AbstractRange;
 import org.xvm.util.S;
 import org.xvm.util.SB;
 
 public class InvokeAST extends AST {
-  String _meth;
+  String _meth, _slice_tmp;
   final boolean _async;
```

Comment:
Can we not have these whitespace diffs? It's a significant part of the pull request, so it's hard to find the actual "payload".  

More importantly, rebasing over a change with lots of whitespace diffs will increase the number of times where git has to conservatively fall back to manual resolution. You are actually the one who has suffered the most because of this. 

I strongly suggest that whatever home rolled solution you use for code formatting auto removes any line with just whitespace in it, replacing it with a line with length 0

Reply:
The change is doing exactly that - removing non-zero length empty lines

Reply:
And Marcus, can we please stop with the hugely judgemental comments.
They are insulting, and are not helping a healthy working environment.
As for "home grown", its the same tool that built all the tools your using in your daily life - its been in widespread use for 40 years.  
Per *YOUR REQUEST* I flipped the "strip trailing whitespace on save" flag, and as a natural consequence every file touched afterwards will 1-time include some WS diffs.

Reply:
Ah! I am sorry! I had it the other way around. The reason I picked this up as whitespaces being introduced rather than removed, was that I remember noticing them being added in your first merge request to master. I then asked if they could be removed before getting merged. I assumed that this was the case. Hence, I misread the diff going the other way while checking the review, and interpreted it that they were somehow back. Thinking that they had been reintroduced there, I pointed it out again, because as we know, white space diffs and git turn automatic rebases into manual ones at too many occasions. Standardising early can actually matter when it comes to avoiding future manual labor.